examples/typescript/aws-s3-static-website/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "cdktf",
+  "version": "1.0.0",
+  "main": "main.js",
+  "types": "main.ts",
+  "license": "MPL-2.0",
+  "private": true,
+  "scripts": {
+    "get": "cdktf get",
+    "build": "tsc",
+    "synth": "cdktf synth",
+    "compile": "tsc --pretty",
+    "watch": "tsc -w",
+    "test": "jest",
+    "test:watch": "jest --watch",
+    "upgrade": "npm i cdktf@latest cdktf-cli@latest",
+    "upgrade:next": "npm i cdktf@next cdktf-cli@next"
+  },
+  "engines": {
+    "node": ">=14.0"
+  },
+  "dependencies": {
+    "@cdktf/provider-aws": "^8.0.10",
+    "cdktf": "^0.11.2",
```

Our examples are currently set to version `0.0.0` to allow our monorepo setup to automatically use the local cdktf package for running the examples.
```suggestion
    "cdktf": "0.0.0",
```